README.md

```diff
@@ -65,6 +67,20 @@ echo '{"name": "B", "version": "14.0.1.0.0", "depends": ["crm"], "license": "Oth
 echo '{"name": "C", "version": "14.0.1.0.0", "depends": ["mail"], "license": "LGPL-3"}' > /tmp/myaddons/c/__manifest__.py
 ```
 
+It is possible to select modified addons compared to a target branch `target`.
+
+```console
+$ manifestoo --select-addons-dir /tmp/myaddons --select-git-modified target list
```

Comment:
`--select-addons-dir` is documenting as selecting all addons in the specified directory. So combining this with `--select-git-modified` probably does not make sense. Maybe we can simply let `--select-git-modified` look in the current directory ? And if later we want to look into multiple directory, add a dedicated option to tell it where to look for git diffs.

Reply:
Sorry, I meant --addons-path instead of --select-addons-dir. Will update.